### PR TITLE
Website: Add note w.r.t. docker container

### PR DESCRIPTION
**Pull request by figitaki**
Adds comment about running in detatched mode.

### CODE REVIEW

File: frontend/website2/pages/docs/getting-started.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default Page({title: "Getting Started"});
 
 <Alert>
 
-  These instructions are tempporarily for a **release-cantidate**. They will be changed on January 25th with the launch of Phase 3.
+  These instructions are tempporarily for a **release-cantidate**. They will be changed on January 24th with the launch of Phase 3.
```
Comment (pranaymohan) [inline frontend/website2/pages/docs/getting-started.mdx]:
```suggestion
  These instructions are temporarily for a **release-candidate**. They will be changed on January 24th with the launch of Phase 3.
```

File: frontend/website2/pages/docs/getting-started.mdx [hunk 2]
```diff
@@ -77,10 +77,10 @@ curl get.docker.com | sh
 Next to download, start and connect to the image by running the command below:
 
 ```
-docker run -it --entrypoint /bin/bash  codaprotocol/coda-daemon:0.0.11-beta1-release-0.0.12-beta-493b4c6
+docker run -it -d --entrypoint /bin/bash  codaprotocol/coda-daemon:0.0.11-beta1-release-0.0.12-beta-493b4c6
```
Comment (pranaymohan) [inline frontend/website2/pages/docs/getting-started.mdx]:
```suggestion
docker run -d --entrypoint /bin/bash  codaprotocol/coda-daemon:0.0.11-beta1-release-0.0.12-beta-493b4c6
```

File: frontend/website2/pages/docs/getting-started.mdx [hunk 3]
```diff
@@ -77,10 +77,10 @@ curl get.docker.com | sh
...
 ```
 
-All the following commands should be run inside the Docker container.
+This starts a new container with the latest release installed and opens a shell. All the following commands should be run inside the Docker container. **NOTE** We are running in detatched mode which means closing the terminal will not stop the docker container.
```
Comment (pranaymohan) [inline frontend/website2/pages/docs/getting-started.mdx]:
```suggestion
This starts a new container with the latest release installed and opens a shell. All the following commands should be run inside the Docker container. **NOTE** We are running in detatched mode which means closing the terminal will not stop the docker container.

If you'd like to run the container interactively, you can add `-it` to the command above. Note that you can always detach the container with the key sequence `ctrl-p` `ctrl-q`.
```

File: frontend/website2/pages/docs/getting-started.mdx (hunk 2 above)
Comment (pranaymohan) [reply frontend/website2/pages/docs/getting-started.mdx]: ignore this one if you want @figitaki

**Comment by bkase** [comment]
Website fails to build here

```
Error: ENOENT: no such file or directory, open './static/img/richardson.jpg'
    at Object.openSync (fs.js:454:3)
    at Object.readFileSync (fs.js:354:35)
    at getHashedPath (/Users/distiller/project/frontend/website/lib/js/src/Links.js:45:22)
    at Array.url (/Users/distiller/project/frontend/website/lib/js/src/Links.js:57:32)
    at TeamSection$Member (/Users/distiller/project/frontend/website/lib/js/src/TeamSection.js:20:39)
    at processChild (/Users/distiller/project/frontend/website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3090:14)
    at resolve (/Users/distiller/project/frontend/website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3013:5)
    at ReactDOMServerRenderer.render (/Users/distiller/project/frontend/website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3436:22)
    at ReactDOMServerRenderer.read (/Users/distiller/project/frontend/website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3395:29)
    at Object.renderToStaticMarkup (/Users/distiller/project/frontend/website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3969:27)
```

**Comment by bkase** [comment]
Outdated now

**Pull request closed without merging**